cf-check/cf-check.c

```diff
@@ -36,7 +37,54 @@ static void print_help()
         "\n");
 }
 
-int main(int argc, char **argv)
+static const char *const CF_CHECK_SHORT_DESCRIPTION =
+    "Utility for diagnosis and repair of local CFEngine databases.";
+
+static const char *const CF_CHECK_MANPAGE_LONG_DESCRIPTION =
+    "cf-check does not evaluate policy or rely on the integrity of the\n"
+    "databases. It is intended to be able to detect and repair a corrupt\n"
+    "database.";
+
+// static const Description COMMANDS[] =
+// {
+//     {"help",     "Prints general help or per topic",
+//                  "cf-check help [command]"},
+//     {"diagnose", "Assess the health of one or more database files",
+//                  "cf-check diagnose"},
+//     {"backup",   "Copy database files to a timestamped folder",
+//                  "cf-check backup"},
+//     {"repair",   "Diagnose, then backup and delete any corrupt databases",
+//                  "cf-check repair"},
+//     {"dump",     "Print the contents of a database file",
+//                  "cf-check dump -a " WORKDIR "/state/cf_lastseen.lmdb"},
+//     {NULL, NULL, NULL}
+// };
+
```

Comment:
Why the commented out COMMANDS[]?

Reply:
It's currently unused, but ready to be used in an improved help menu or man page.